tests/test_datetimes.py

```diff
@@ -628,6 +628,16 @@ def test_time_with_nested():
         assert time() == second
 
 
+@pytest.fixture
+def change_stack_limit():
+    ori_val = api.call_stack_inspection_limit
+    api.call_stack_inspection_limit = 100  # just to increase coverage
+    yield
+    # Restore to normal after test(s)
+    api.call_stack_inspection_limit = ori_val
+
+
+@pytest.mark.usefixtures('change_stack_limit')
 def test_should_use_real_time():
```

Comment:
`test_should_use_real_time` fails. But if I comment out the lines that change `call_stack_inspection_limit`, they pass.

As a sanity check, I undid my change, left the `call_stack_inspection_limit` lines commented out, and the tests still passed. So I'm not sure what this is doing, other than the comment about increasing coverage.

I think the other tests failed (from the run from the first commit) because the limit was 'leaking' across the tests. The fixture I added would fix this, but perhaps these lines should just be removed?